### PR TITLE
Features/variation ftp sync

### PR DESCRIPTION
**Pull request by marcoooo**
Added first draft (not really configurable) script to sync some files from there source specified in `ensembl_genome_metadata` db to target, say ftp `/nfs`

## Description

fetch dataset source information from dataset type. then sync the files in the `tgt-root` specie

## Use case

First use case is to copy variation files submitted to us to target nfs. 

## Benefits

First piece of code for release processing on Variation Datasets. 

## Possible Drawbacks

None to my knowledge.

## Testing

- [ ] Have you added/modified unit tests to test the changes?
- [ ] If so, do the tests pass?
- [ ] Have you run the entire test suite and no regression was detected?
- [ ] TravisCI passed on your branch

Dependencies
------------

> If applicable, define what code dependencies were added and/or updated.

### CODE REVIEW

File: requirements.in [hunk 1]
```diff
@@ -2,4 +2,4 @@ requests>=2.25.0,<3
 pydantic~=1.10.5
 graphene~=2.1.9
 graphene-sqlalchemy~=2.3.0
-git+https://github.com/Ensembl/ensembl-metadata-api.git@2.0.1a2#egg=ensembl-metadata-api
+git+https://github.com/Ensembl/ensembl-metadata-api.git@2.1.0.dev1#egg=ensembl-metadata-api
```
Comment (vinay-ebi) [inline requirements.in:5]: installing dev version of metadata api

File: src/python/scripts/genome_info.py [hunk 2]
```diff
@@ -19,77 +19,81 @@
 '''
 Fetch Genome Info from the new metadata api
```
Comment (vinay-ebi) [inline src/python/scripts/genome_info.py:20]: This script is obsolete we can remove it

File: requirements.in (hunk 1 above)
Comment (marcoooo) [reply requirements.in:5]: Yeah, I am awaiting the api PR to be merge to do an actual version. But I need this to use the ReleaseStatus and DatasetStatus constants.

**Review by vinay-ebi — APPROVED** [review]

File: requirements.in (hunk 1 above)
Comment (marcoooo) [reply requirements.in]: Just updated to a more stable one (a1)

**Pull request merged**